chains/fantomopera/metadata.yaml

```diff
@@ -0,0 +1,28 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://api.ftmscan.com/api
+    family: etherscan
+    name: FTMScan
+    url: https://ftmscan.com
+chainId: 250
+deployer:
+  name: Abacus Works
+  url: https://www.hyperlane.xyz
```

Comment:
please use your own deployer details

Reply:
I'm just a regular user, what should I use ? Should I use my Github account or my email address ?

Reply:
your github is probably good

you could also remove the `addresses` and omit the `deployer` details

Reply:
What Paul means is, please either remove the addresses.yaml file (the artifacts from the deployment) or add a `deployer` section to the metadata file. If there's a deployment in the registry, we'd like to also have some basic information about who deployed it.